client/landing/subscriptions/components/site-subscription-page/site-subscription-page.tsx

```diff
@@ -76,19 +92,49 @@ const SiteSubscriptionPage = () => {
 						<FormattedHeader brandFont headerText={ siteName } subHeaderText={ subHeaderText } />
 					</header>
 
-					<SiteSubscriptionSettings
-						blogId={ blogId }
-						notifyMeOfNewPosts={ notifyMeOfNewPosts }
-						emailMeNewPosts={ emailMeNewPosts }
-						deliveryFrequency={ deliveryFrequency }
-						emailMeNewComments={ emailMeNewComments }
-					/>
+					{ unsubscribed && (
+						<Notice type={ NoticeType.Success }>
+							{ translate(
+								'You have successfully unsubscribed and will no longer receive emails from %s.',
+								{
+									args: [ data.siteName ],
+									comment: 'Name of the site that the user has unsubscribed from.',
+								}
+							) }
+						</Notice>
+					) }
+
+					{ unsubscribeError && (
+						<Notice type={ NoticeType.Error }>
+							{ translate( 'There was an error when trying to unsubscribe from %s.', {
```

Comment:
Hey @saygunnyc 👋🏼 Do we have a preferred copy that should display when unsubscribing fails for some reason?

The text you can see below is just temporary I have used so far:

![Markup on 2023-05-17 at 12:01:28](https://github.com/Automattic/wp-calypso/assets/25105483/a17b927e-b7f7-4b54-8ea8-cd4dc7185dec)

Thank you!